src/WebForms/WebForms.csproj

```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="System.CodeDom" Version="9.0.3" />
     <PackageReference Include="Microsoft.AspNetCore.SystemWebAdapters" Version="2.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.SystemWebAdapters.CoreServices" Version="2.0.0" />
+    <PackageReference Include="System.Drawing.Common" Version="9.0.3" />
```

Comment:
We had removed this before - what's requiring it? I'm fine adding it back but it would be nice to decouple it if possible